src/main/java/org/jetbrains/kotlinx/lincheck/strategy/managed/ManagedCTestConfiguration.kt

```diff
@@ -49,6 +50,9 @@ abstract class ManagedCTestConfiguration(
             // Non-determinism should not be present in managed executions, but luckily the classes
             // can be just ignored, so that no thread context switches are added inside their methods.
             forClasses("kotlinx.coroutines.internal.StackTraceRecoveryKt").allMethods().ignore(),
+            // Ignore the DispatchedTask methods, they can be called by Lincheck.
+            @Suppress("INVISIBLE_REFERENCE", "INVISIBLE_MEMBER")
+            forClasses(DispatchedTask::class.qualifiedName!!).allMethods().ignore(),
```

Comment:
That won't work.
`ManagedStrategyGuarantees` are supported only for methods that are called from the transformed code, because we add strategy invocation before and after the method's invocation.
See `ManagedStrategyTransformer.CancellabilitySupportMethodTransformer` to see how `CancellableContinuationImpl.cancel` из handled. We either need to handle `DispatchedTask` in the same way, or add a way to call `strategy.enterIgnoredSection` and `strategy.leaveIgnoredSection` directly in the `cancelByLincheck` method.

Note that if exceptions are possible, then try-catch block should be added in the transformed code